save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/VulnerabilitiesFiltersRow.kt

```diff
@@ -35,46 +29,65 @@ val vulnerabilitiesFiltersRow: FC<NameFilterRowProps> = FC { props ->
             }
             div {
                 className = ClassName("row")
-                div {
-                    className = ClassName("col-auto align-self-center")
-                    +"Name: "
-                }
                 div {
                     className = ClassName("col-auto mr-3")
                     input {
                         type = InputType.text
                         className = ClassName("form-control")
-                        value = filtersName ?: ""
+                        value = filter.prefixName
+                        placeholder = "Name..."
                         required = false
-                        onChange = {
-                            setFiltersName(it.target.value)
+                        onChange = { event ->
+                            setFilter { oldFilter ->
+                                oldFilter.copy(prefixName = event.target.value)
+                            }
+                        }
+                    }
+                }
+                div {
+                    className = ClassName("col-auto")
+                    inputWithDebounceForString {
+                        selectedOption = tagPrefix
+                        setSelectedOption = { setTagPrefix(it) }
+                        getUrlForOptionsFetch = { prefix -> "$apiUrl/tags/vulnerabilities?prefix=$prefix" }
+                        placeholder = "Tag..."
```

Comment:
Up to you, but I'd rather use a single-character ellipsis, there's even a designated HTML entity `&hellip;`

Reply:
The point is that this symbol is very hard to read in IDEA, so I would leave as is